packages/db/src/lib/granule.ts

```diff
@@ -19,8 +19,8 @@ import { UpdatedAtRange } from '../types/record';
 const { TableNames } = require('../tables');
 
 export const getGranuleCollectionId = async (
-  knexOrTransaction: Knex | Knex.Transaction,
-  granule: PostgresGranule | PostgresGranuleRecord
+  knexOrTransaction: Knex,
+  granule: { collection_cumulus_id: number }
```

Comment:
While I don't functionally disagree with constraining the type in *this* case, the proposed change is again a breaking change from what was in the file prior.   

 We should consider this in main, but this changeset is being proposed against the 11.1.x baseline and given it's a public API I think we should revert the typing update here.   Additionally it's inconsistent with our current usage of it in places like `granule-remove-from-cmr.js`.... which raises the question of why the typedef/import is passing the check at all 👀